python/sglang/srt/sampling/sampling_params.py

```diff
@@ -14,9 +14,16 @@
 """Sampling parameters for text generation."""
 
 import logging
-import sre_parse
 from typing import Any, Dict, List, Optional, Union
 
+# Use re._parser instead of deprecated sre_parse module.
+# Note that `re._parser` is a private API and may change without warning in future Python versions.
+try:
+    from re import _parser as sre_parse
```

Comment:
qq can you clarify the specific python version

Reply:
From Python 3.11, added this info in title and code